modules/rostests/apitests/shell32/ShellExecuteEx.cpp

```diff
@@ -81,8 +155,67 @@ TestShellExecuteEx(const WCHAR* Name, BOOL ExpectedResult)
     }
 }
 
+static
+VOID
+TestShellExecuteEx1(const WCHAR* Name, const WCHAR* Params)
+{
+    SHELLEXECUTEINFOW ShellExecInfo;
+    BOOL Result;
+    CHAR OSVendor[100] = {0};
+    INT iResult;
+    UINT_PTR retval = SE_ERR_NOASSOC;
+
+    ZeroMemory(&ShellExecInfo, sizeof(ShellExecInfo));
+    ShellExecInfo.cbSize = sizeof(ShellExecInfo);
+    ShellExecInfo.fMask = SEE_MASK_NOCLOSEPROCESS | SEE_MASK_FLAG_NO_UI;
+    ShellExecInfo.hwnd = NULL;
+    ShellExecInfo.nShow = SW_SHOWNORMAL;
+    ShellExecInfo.lpFile = Name;
+    ShellExecInfo.lpParameters = Params;
+    ShellExecInfo.lpDirectory = NULL;
+    ShellExecInfo.lpVerb = L"open";
+
+    iResult = GetOSVendor(OSVendor, sizeof(OSVendor));
+    printf("OSVendor is '%s'.\n", OSVendor);
+
+    if(strcmp(OSVendor, "Microsoft") == 0)
+        skip("Test does not work on MS Windows.\n");
```

Comment:
`skip()` without `return`?

Reply:
@SergeGautherie, Yes, I am using skip() without return's. For an existing example look at this:
modules\rostests\winetests\xinput1_3\xinput.c. Here there are five (5) skip() statements and no return statements. Others are readily found by just looking. I leave it as an exercise for the reader to determine why you want each of these different usage styles.